mne/evoked.py

```diff
@@ -312,7 +312,7 @@ def plot_image(self, picks=None, exclude='bads', unit=True, show=True,
     def plot_topo(self, layout=None, layout_scale=0.945, color=None,
                   border='none', ylim=None, scalings=None, title=None,
                   proj=False, vline=[0.0], fig_background=None,
-                  merge_grads=False, legend=True, axes=None,
+                  merge_channels=False, legend=True, axes=None,
```

Comment:
you cannot change public APIs like that. We need deprecation cycles.

Reply:
To be honest I would revert this one unless you need it already for NIRS

Reply:
No don't need this at all. I just misunderstood a previous comment and thought I was meant to change all occurrences of merge grad to merge channels. I'll change this back